tests/test_integrate.py

```diff
@@ -182,3 +182,14 @@ def test_log(loglines):
         db = sqlite3.connect(totalsdb)
         rows_no = db.execute("select count(*) from countme_totals;").fetchone()[0]
         assert int(rows_no) > 0
+
+        # Test with csv_dump option
+        csv_dump = open(f"{tmp_dir}/test.csv", "w+")
+        totals(
+            countme_totals=totalsdb,
+            countme_raw=rawdb,
+            progress=False,
+            csv_dump=csv_dump,
+        )
+        csv_dump.seek(0)
+        assert len(csv_dump.readlines()) > 0
```

Comment:
The second call to `totals()` also writes the sqlite DB, so both can be rolled into one:

```
        csv_dump = open(f"{tmp_dir}/test.csv", "w+")
        totals(
            countme_totals=totalsdb,
            countme_raw=rawdb,
            progress=False,
            csv_dump=csv_dump,
        )
        db = sqlite3.connect(totalsdb)
        rows_no = db.execute("select count(*) from countme_totals;").fetchone()[0]
        assert int(rows_no) > 0
        csv_dump.seek(0)
        assert len(csv_dump.readlines()) > 0
```

I'll commit this into your branch.